### PR TITLE
Fix documentation comments

### PR DESCRIPTION
**Pull request by liamwhite**
Fixes the wording of a few documentation comments to the way I believe they were intended to be written.

### CODE REVIEW

File: priv/templates/phx.gen.auth/notifier.ex [hunk 1]
```diff
@@ -32,7 +32,7 @@ defmodule <%= inspect context.module %>.<%= inspect schema.alias %>Notifier do
   end
 
   @doc """
-  Deliver instructions to reset password account.
+  Deliver instructions to reset account password.
```
Comment (aaronrenner) [inline priv/templates/phx.gen.auth/notifier.ex]:
```suggestion
  Deliver instructions to reset a <%= schema.singular %> password.
```

File: priv/templates/phx.gen.auth/notifier.ex [hunk 2]
```diff
@@ -52,7 +52,7 @@ defmodule <%= inspect context.module %>.<%= inspect schema.alias %>Notifier do
   end
 
   @doc """
-  Deliver instructions to update your email.
+  Deliver instructions to update account email.
```
Comment (aaronrenner) [inline priv/templates/phx.gen.auth/notifier.ex]:
```suggestion
  Deliver instructions to update a <%= schema.singular %> email.
```

**Review by aaronrenner — COMMENTED** [review]
Thanks for finding these! A couple of suggestions... WDYT?

**Comment by liamwhite** [comment]
LGTM

**Pull request merged**